packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -528,6 +525,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
       let newItems: T[] = this.state.items.concat([processedItemObject]);
       this._updateSelectedItems(newItems);
     }
+    this.setState({ suggestedDisplayValue: '' });
```

Comment:
Housekeeping. It ensures that when you click out of the picker that and it queues up an item to autoselect that it won't show incorrect text briefly.